src/js/services/forms.js

```diff
@@ -52,11 +52,13 @@ export default App.extend({
     const updated = dayjs().format();
     try {
       store.set(this.getStoreId(), { submission, updated });
+      this.trigger('submissionUpdated', updated);
     } catch (e) /* istanbul ignore next: Tested locally, test runner borks on CI */ {
       store.each((value, key) => {
         if (String(key).startsWith('form-subm-')) store.remove(key);
       });
       store.set(this.getStoreId(), { submission, updated });
+      this.trigger('submissionUpdated', updated);
```

Comment:
Do we need this one?

Reply:
nope this section is ignored in tests because cypress can't do it.  we probably don't actually need to do this though.. it's an edge case, and it will fix itself on the next use interaction with the form.  Let's not add this.

Reply:
Gotcha, removed in the last push.